pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/blocks/TransferableBlock.java

```diff
@@ -76,7 +83,7 @@ public DataSchema getDataSchema() {
    *
    * @return data container.
    */
-  public List<Object[]> getContainer() {
```

Comment:
@agavra this involved changing the contain API from List to Collection. to support PriorityQueue pre-sorted intermediate result blocks.

Reply:
actually nvm. with the leaf node split and all. we might not want to change this API and instead we should ask the orderby combine operator to flatten the priorityqueue into a list. 
1. it won't be preserving the sorted structure anyway if transfer over the wire
2. the whole idea is to avoid extra memory usage not save time, but it seems like we have to do it anyway, with split block and all